.github/workflows/ci-tests-data.yml

```diff
@@ -94,7 +94,12 @@ jobs:
 
       - name: Testing Data
         working-directory: tests/tests_data
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_PUB_ACCESS_KEY }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_PUB_SECRET_KEY }}
+          AWS_DEFAULT_REGION: ${{ secrets.AWS_REGION }}
```

Comment:
What is the user and permissions? 
Pls, reach me for a validation vulnerability check

Reply:
The bucket is public
Just need to be auth to AWS
Any dev can contribute if they use their own credentials

Reply:
removed.